### PR TITLE
Fix/553 simplified attributes should be summarised

### PR DESCRIPTION
**Pull request by sahand-asgarpour**
## Issue addressed
Solves #553 

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [x] I HAVE discussed my solution with (other) members of the RA2CE team.

## What has been done?
Explain how you addressed the resolution of the related issue, what choices you made and why.

### Checklist
- [x] Code is formatted using our custom `black` and `isort` definitions.
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: ra2ce/network/network_simplification/nx_to_snkit_network_converter.py [hunk 1]
```diff
@@ -49,6 +51,7 @@ def convert(self) -> SnkitNetwork:
         """
         # Extract graph values
         _crs = self.networkx_graph.graph.get("crs", None)
+        self.networkx_graph = add_x_y_to_nodes(self.networkx_graph)
```
Comment (ArdtK) [inline ra2ce/network/network_simplification/nx_to_snkit_network_converter.py:54]: What is the reason for and effect of this change?

File: ra2ce/network/network_simplification/snkit_network_wrapper.py
Comment (ArdtK) [inline ra2ce/network/network_simplification/snkit_network_wrapper.py:1]: Functions `_aggfunc_with_demand_edge` and `_aggfunc_no_demand_edge` contain quite some code duplication. Is there a way to reduce this?
A simple solution could be passing a flag `with_demand` to it, being `true` or `false`. A more OO-like solution can be considered as well.

**Review by ArdtK — COMMENTED** [review]
A questions and suggestion

File: ra2ce/network/network_simplification/nx_to_snkit_network_converter.py (hunk 1 above)
Comment (sahand-asgarpour) [reply ra2ce/network/network_simplification/nx_to_snkit_network_converter.py:55]: I added a general comment. Can not recall its relation to the issue. Tried to check it by commenting this line. Tests passed.

File: ra2ce/network/network_simplification/snkit_network_wrapper.py
Comment (sahand-asgarpour) [reply ra2ce/network/network_simplification/snkit_network_wrapper.py:1]: Made a function using a flag.

**Pull request merged**